.github/workflows/test.yaml

```diff
@@ -84,7 +84,7 @@ jobs:
     - name: Run spread tests
       env:
           SPREAD_GOOGLE_KEY: ${{ secrets.SPREAD_GOOGLE_KEY }}
-      run: /tmp/spread -v google-unstable:${{ matrix.system }}:tests/...
+      run: /tmp/spread -abend -v google-unstable:${{ matrix.system }}:tests/...
```

Comment:
I think we said we will run without "-v" but not quite yet with "-abend", let's see how readable the (error) output is without -v first before we go into fail fast mode (which has it's own drawbacks).

Reply:
Can we have -abend on unstable and no -abend on the rest as a compromise?

Reply:
Can you please remove -v still to see how this looks in the output?

Reply:
+1